.github/workflows/stale.yml

```diff
@@ -14,22 +14,33 @@ jobs:
       - uses: actions/stale@6f05e4244c9a0b2ed3401882b05d701dd0a7289b # pin@v7
         with:
           # Idle number of days before marking issues stale, set `-1` to disable
-          days-before-issue-stale: 180
+          days-before-issue-stale: 60
 
           # Idle number of days before marking issues stale, set `-1` to disable
-          days-before-pr-stale: -1
+          days-before-pr-stale: 60
 
           # Idle number of days before closing stale issues/PRs
-          days-before-close: 14
+          days-before-close: 15
 
           # Comment on the staled issues
-          stale-issue-message: "This issue has been automatically marked as stale because it has not had recent activity. This is for us to prioritize issues that are still relevant to our community. It will be closed if no further activity occurs within the next 14 days. If this issue is still relevant to you, please leave a comment."
+          stale-issue-message: "This issue has been marked as stale because it requires further input but has not seen activity in the past months. This is for us to prioritize issues that are still relevant and actionable. It will be closed if no further activity occurs within the next 15 days. If this issue is still relevant to you, please help us in gathering the necessary input."
+
+          # Comment on the staled PRs
+          stale-pr-message: "This PR has been marked as stale because it requires further changes but has not seen activity in the past months. This is for us to prioritize PRs that can be reviewed and merged. It will be closed if no further activity occurs within the next 15 days. If you still have interest in this PR, please help us finalizing it."
```

Comment:
```suggestion
          stale-pr-message: "This PR has been marked as stale because it requires further changes but has not seen activity in the past months. This is for us to prioritize PRs that can be reviewed and merged. It will be closed if no further activity occurs within the next 15 days. If you still have interest in this PR, please help us finalizing it. Please let us know in case you are stuck on the required changes."
```